data/json/flags.json

```diff
@@ -83,6 +83,18 @@
     "info": "As a weapon, this item is <good>well-made</good> and will <info>withstand the punishment of combat</info>.",
     "conflicts": [ "FRAGILE_MELEE" ]
   },
+  {
+    "id": "EATEN_COLD",
+    "type": "json_flag",
+    "context": [ "COMESTIBLE" ],
+    "info": "This item tastes <good>better</good> while <info>cold</info>."
```

Comment:
This might sound better if you remove "item" from the sentence: `This tastes <good>better</good> while <info>cold</info>.`

You can also use `<color_light_cyan>cold</color>`. For a list of valid colors, [check here](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/COLOR.md).